python/thrift/appengine-ssl-gateway/.editorconfig

```diff
@@ -0,0 +1,19 @@
+# For more info, see http://EditorConfig.org
```

Comment:
`.editorconfig` helps everyone contributing to a project have the same style, this is not a per-person configuration—it's not auto-generated and should be versioned, not ignored.

I think we should actually move this to the root directory of the repo but I didn't want to impose this style for all the files in the repo before checking with you and others.

I can take it out and we can submit this file separately as it's not crucial for this change.